Fix build error when CONFIG_PM_SLEEP is not set

--- a/drivers/pci/host/pci-imx6.c	2017-01-14 19:50:14.159715318 +0100
+++ b/drivers/pci/host/pci-imx6.c	2017-01-14 19:50:42.182645605 +0100
@@ -1311,7 +1311,9 @@ static struct platform_driver imx6_pcie_
 		.name	= "imx6q-pcie",
 		.owner	= THIS_MODULE,
 		.of_match_table = imx6_pcie_of_match,
+#ifdef CONFIG_PM_SLEEP
 		.pm = &pci_imx_pm_ops,
+#endif
 	},
 	.shutdown = imx6_pcie_shutdown,
 };

